defopt.py

```diff
@@ -118,8 +118,23 @@ def _create_parser(funcs, *args, **kwargs):
     return parser
 
 
-class _Formatter(RawTextHelpFormatter, ArgumentDefaultsHelpFormatter):
-    pass
+class _Formatter(RawTextHelpFormatter):
+
+    # modified from ArgumentDefaultsHelpFormatter to add type information, and
+    # remove defaults for varargs (`ZERO_OR_MORE`).
+    def _get_help_string(self, action):
+        info = []
+        if action.type is not None and '%(type)' not in action.help:
+            info.append('%(type)s')
+        if (action.default != argparse.SUPPRESS
```

Comment:
The real implementation of this compares to `SUPPRESS` using `is not`, and I'd be inclined to do the same. I'd also leave a short comment about why the handling of `OPTIONAL` is being ignored here (presumably because we never use it so it doesn't matter).